server/src/main/java/org/apache/seata/server/coordinator/DefaultCoordinator.java

```diff
@@ -414,8 +425,7 @@ protected void handleRetryCommitting() {
         SessionHelper.forEach(committingSessions, committingSession -> {
             try {
                 // prevent repeated commit
-                if ((GlobalStatus.Committing.equals(committingSession.getStatus())
-                        || GlobalStatus.Committed.equals(committingSession.getStatus()))
+                if (GlobalStatus.Committed.equals(committingSession.getStatus())
                         && !committingSession.isDeadSession()) {
```

Comment:
不存在Committed状态且isDeadSession为false的情况吗？如果是这样，那应该移除整个if判断。
Is there no such thing as a Committed state and isDeadSession is false? If so, then the entire if judgment should be removed.

Reply:
> 不存在Committed状态且isDeadSession为false的情况吗？如果是这样，那应该移除整个if判断。 Is there no such thing as a Committed state and isDeadSession is false? If so, then the entire if judgment should be removed.

isDeadSession只是为了committing和rollbacking这两个状态，并且能走到handleRetryCommitting中时，必定已经是已经判断过isDeadSession，然后才转变的状态，committed状态仅在异步线程中变化，同步二节点处理不会转变为committed。
isDeadSession is only for the two states of committing and rollbacking, and when it can go to the handleRetryCommitting, it must have already judged the isDeadSession, and then transitioned to the state. The committed state only changes in the asynchronous thread, and the synchronous two-node processing will not change to committed.